core/core.go

```diff
@@ -210,6 +248,46 @@ func (n *IpfsNode) OnlineMode() bool {
 	return n.onlineMode
 }
 
+func (n *IpfsNode) Bootstrap(ctx context.Context, peers []peer.PeerInfo) error {
+	if n.DHT != nil {
```

Comment:
not sure if this is still WIP, but the "connecting to bootstrap peers" part should be in this function too.